spec/supplejack/concept_spec.rb

```diff
@@ -128,5 +128,12 @@ module Supplejack
         end
       end
     end
+
+    describe '#all' do
+      it 'calls the get method' do
```

Comment:
I am against writing old syntax just because the rest of the file is like that.

I would take 15minutes to learn the new syntax and update the rest of the file to use the new syntax.

but it's your call on if you want to do that.